src/src/rx_main.cpp

```diff
@@ -648,7 +650,7 @@ void loop()
     //crsf.RXhandleUARTout(); using interrupt based printing at the moment
 
     #if defined(PLATFORM_ESP8266) && defined(AUTO_WIFI_ON_BOOT)
-    if ((connectionState == disconnected) && !webUpdateMode && millis() > 20000 && millis() < 21000 && LastSyncPacket == 0)
+    if ((connectionState == disconnected) && !webUpdateMode && millis() > 20000 && !disableWebServer)
```

Comment:
If you put the !disableWebServer first the compiler can short-circuit the rest of the conditional once the flag has been set and won't have to make the millis() call. Not going to be performance critical here, but if you get into the habit of thinking about short circuiting logical expressions it can pay dividends throughout the code.